test/test_data/ffmpeg.py

```diff
@@ -4,7 +4,7 @@
         "version": "4.1.3",
         "version_strings": [
             "%s version 4.1.3",
-            "Codec %s is not recognized by FFmpeg.",
+            "Codec '%s' is not recognized by FFmpeg.",
```

Comment:
In future, can you put unrelated changes to checkers in a separate PR?  They could all be grouped together, but they shouldn't be grouped with the InputEngine changes.  It's considered bad form by most open source projects to group them the way you have.

Reply:
I have put it here because I have to make VersionScanner generator to use InputEngine parsed data as supplement and it was breaking the test_scanner because I have also included test for contains string previously but I have forget to clean up scanner and test was only passing due to previous scan has detected it. So, that's why when I converted that in generator this bug come to surface.  I should have divide this PR in two parts: 1) Making version_scanner generator and 2) building input_engine.